cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -1392,4 +1392,198 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
     cy.wait('@refreshCellQuery')
     cy.wait('@refreshSecondQuery')
   })
+
+  it('can pause a cell, which takes it (and only it) out of auto refresh loop, even if second cell has same query', done => {
+    cy.get('@org').then(({id: orgID, name}: Organization) => {
+      cy.createDashboard(orgID).then(({body}) => {
+        cy.fixture('routes').then(({orgs}) => {
+          cy.visit(`${orgs}/${orgID}/dashboards/${body.id}`)
+          cy.getByTestID('tree-nav')
+        })
+      })
+      cy.window().then(win => {
+        win.influx.set('pauseCell', true)
+        win.influx.set('newAutoRefresh', true)
```

Comment:
Might need to add a wait for one second here. There’s flakiness around feature flag setting in cypress tests